ui/pages/first-time-flow/select-action/index.scss

```
@@ -34,7 +35,7 @@
     border: 1px solid #d8d8d8;
     box-sizing: border-box;
     border-radius: 10px;
-    margin-left: 22px;
+    padding: 10px;
```

Adhere to `8px` grid 
```suggestion
    padding: 8px;
```